tools/bamtofastq/2.0.0/bamtofastq__2.0.0.cwl

```diff
@@ -0,0 +1,322 @@
+    inputBinding:
+      prefix: S=
+      separate: false
+  unmatched_pairs_first_mate:
+    label: unmatched pairs first mate
+    type: string?
+    doc: "unmatched pairs first mates"
+    inputBinding:
+      prefix: O=
+      separate: false
+  unmatched_pairs_second_mate:
+    label: unmatched pairs mate2
+    type: string?
+    doc: "unmatched pairs second mates"
+    inputBinding:
+      prefix: O2=
+      separate: false
+  collate:
+    label: collate
+    type: int?
+    doc: "collate pairs"
+    inputBinding:
+      prefix: collate=
+      separate: false
+      valueFrom: "$(boolean_to_int(self))"
+  combs:
+    label: combs
+    type: boolean?
+    doc: "print some counts after collation based processing"
+    inputBinding:
+      prefix: combs=
+      separate: false
+      valueFrom: "$(boolean_to_int(self))"
+  filename:
+    label: file name
+    type: File
+    doc: "input filename"
+    streamable: true
+    inputBinding:
+      prefix: filename=
+      separate: false
+  input_format:
+    label: input format
+    type: string?
+    doc: "input format: cram, bam or sam [bam]"
+    inputBinding:
+      prefix: inputformat=
+      separate: false
+  reference:
+    label: reference
+    type: File?
+    doc: "name of reference FastA in case of inputformat=cram"
+    inputBinding:
+      prefix: reference=
+      separate: false
+  ranges:
+    label: ranges
+    type: string?
+    doc: "input ranges (bam and cram input only, default: read complete file)"
+    inputBinding:
+      prefix: ranges=
+      separate: false
+  exclude:
+    label: exclude
+    type: string?
+    doc: "exclude alignments matching any of the given flags [SECONDARY,SUPPLEMENTARY]"
+    inputBinding:
+      prefix: exclude=
+      separate: false
+  disable_validation:
+    label: disable validation
+    type: boolean?
+    doc: "disable validation of input data [0]"
+    inputBinding:
+      prefix: disablevalidation=
+      separate: false
+      valueFrom: "$(boolean_to_int(self))"
+  colh_log:
+    label: col h log
+    type: int?
+    doc: "base 2 logarithm of hash table size used for collation [18]"
+    inputBinding:
+      prefix: colhlog=
+      separate: false
+  cols_bs:
+    label: cols bs
+    type: int?
+    doc: "size of hash table overflow list in bytes [33554432]"
+    inputBinding:
+      prefix: colsbs=
+      separate: false
+  tmp_file_name:
+    label: t
+    type: string?
+    doc: "temporary file name [bamtofastq_*]"
+    inputBinding:
+      prefix: exclude=
+      separate: false
+  gzip:
+    label: gz
+    type: boolean?
+    doc: "compress output streams in gzip format (default: 0)"
+    inputBinding:
+      prefix: gz=
+      separate: false
+      valueFrom: "$(boolean_to_int(self))"
+  level:
+    label: level
+    type: int?
+    doc: "compression setting if gz=1 (-1=zlib default,0=uncompressed,1=fast,9=best)"
+    inputBinding:
+      prefix: level=
+      separate: false
+  fasta:
+    label: fasta
+    type: int?
```

Comment:
should be boolean

Reply:
https://github.com/umccr/cwl-ica/commit/8b05df6903d1b507342b1b30cf9c0846c3c38396